patches/core/inox-patchset/0001-fix-building-without-safebrowsing.patch

```diff
@@ -332,7 +332,7 @@
        download_->GetURL(), download_->GetContentDisposition(), referrer_charset,
        suggested_filename, sniffed_mime_type, default_filename);
  
--  // We don't replace the file extension if safe browsing consider the file
+-  // We don't replace the file extension if sfafe browsing consider the file
```

Comment:
Huh, I don't think I've seen a typo added on accident before. Interesting

Reply:
They might've just reverted some nasty commits, one of which supposed to correct the spelling :)